src/content/okteto-license.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Okteto License
+description: This page provides information about how Okteto's license works for the products we provide.
+sidebar_label: Okteto License
+id: okteto-license
+---
+
+Okteto requires a license key to use any of our paid products. This document explains how our license works for the different products we provide to remove any uncertainty about feature or usage limitations during a trial period or for your chosen tier of paid service.
+
+# Why a license is required
+
+We require you obtain a license to use Okteto because the product is highly customizable and we want to ensure you have a great experience getting it installed and configured. Obtaining a license from us kicks off specific customer success workflows to help us provide you with the right guidance and support.
+
+Don't worry. We don't require you speak with someone from our Sales team to try Okteto or deal with more friction than is necessary. We want you to experience Okteto in the ways we believe will inform your decisions and requiring a license is the first step on that journey.
+
+This requirement also helps us know who's using our product for typical product development and marketing initiatives that enable us to continue building more and better features.
+
+# How to obtain a license
+
+There are two ways to obtain a license from Okteto.
+
+1. Fill out the self-hosted [free trial form](https://www.okteto.com/free-trial/) on our website
```

Comment:
Outside of the scope of the PR, but do we need to explain why there's no 'get free license' path and only trial?

Reply:
We could. In my opinion I think our docs could use an overhaul of the `Overview` page. Or maybe create a similar one but for the "business logistics" of trying Okteto.

The current `Overview` page summarizes the main features and experiences of Okteto. Maybe we need another, similar page to talk about things like this: why a license is required, how to obtain and use one, why we don't offer a free license, etc.

But for now I think it's implied why we don't offer a free license in the intro paragraph and I'm supportive of being more explicit in a v2 of this page or as part of what I'm proposing earlier in this comment 👍